src/pages/docs/Connecting to Databases/sql-server.mdx

```diff
@@ -6,12 +6,12 @@ index: 24
 version: 1
 ---
 
-## Snowflake
+## Sql Server
```

Comment:
I think sql has to be all caps, this is more a trademark thing? 

"SQL Server" - https://www.microsoft.com/en-us/sql-server/sql-server-2019